cmd/ronin/dbcmd.go

```diff
@@ -243,8 +250,86 @@ WARNING: This is a low-level operation which may cause database corruption!`,
 		},
 		Description: "Exports the specified chain data to an RLP encoded stream, optionally gzip-compressed.",
 	}
+	dbInspectEnodeDBCmd = &cli.Command{
+		Action: dbInspectEnodeDB,
+		Name:   "inspect-enodedb",
+		Usage:  "Inspect nodes in enode db",
+		Flags: []cli.Flag{
+			utils.DataDirFlag,
+			&cli.StringFlag{
+				Name:  "peersfile",
+				Usage: "File containing a list of peers from admin.peers",
+			},
+		},
+		Category: "DATABASE COMMANDS",
+	}
 )
 
+func dbInspectEnodeDB(ctx *cli.Context) error {
+	path := ctx.String(utils.DataDirFlag.Name)
+	db, err := enode.OpenDB(path)
+	if err != nil {
+		return err
+	}
+
+	nodeCount := 0
+	unknownNodeCount := 0
+	inspectedNodes := make(map[string]*enode.Node)
+	unknownNodes := []*enode.Node{}
+	db.IterateNodes(func(n *enode.Node) error {
+		nodeCount++
+		var eth struct {
+			ForkID forkid.ID
+			Tail   []rlp.RawValue `rlp:"tail"`
+		}
+		if n.Record().Load(enr.WithEntry("eth", &eth)) == nil {
+			log.Info("Node", "ID", n.ID(), "IP", n.IP(), "UDP", n.UDP(), "TCP", n.TCP(), "eth", eth)
+		} else {
+			unknownNodeCount++
+			unknownNodes = append(unknownNodes, n)
+		}
+		inspectedNodes[n.ID().String()] = n
+		return nil
+	})
+
+	for _, n := range unknownNodes {
+		log.Info("Unkown node", "ID", n.ID(), "IP", n.IP(), "UDP", n.UDP(), "TCP", n.TCP())
+	}
+	log.Info("Total nodes", "count", nodeCount, "unknown", unknownNodeCount)
+
+	if ctx.IsSet("peersfile") {
+		f, err := os.Open(ctx.String("peersfile"))
+		if err != nil {
+			return err
+		}
+
+		peersInfo := []*p2p.PeerInfo{}
+		decoder := json.NewDecoder(f)
+		err = decoder.Decode(&peersInfo)
+		if err != nil {
+			return err
+		}
+
+		foundInEnodeDB := 0
+		for _, peerInfo := range peersInfo {
+			if peerInfo.Network.Inbound {
```

Comment:
Please add comment for this check here or above `if ctx.IsSet("peersfile")` to describe the purpose